sql/core/src/test/scala/org/apache/spark/sql/connector/KeyGroupedPartitioningSuite.scala

```diff
@@ -1169,7 +1169,6 @@ class KeyGroupedPartitioningSuite extends DistributionAndOrderingSuiteBase {
     Seq(true, false).foreach { shuffle =>
       withSQLConf(
         SQLConf.V2_BUCKETING_SHUFFLE_ENABLED.key -> shuffle.toString,
-        SQLConf.V2_BUCKETING_PUSH_PART_VALUES_ENABLED.key -> "true",
```

Comment:
Please revert all test case changes from this PR, @szehon-ho .

Reply:
OK